doctr/datasets/cord.py

```diff
@@ -97,7 +97,9 @@ def __init__(
                 crops = crop_bboxes_from_image(img_path=os.path.join(tmp_root, img_path),
                                                geoms=np.asarray(box_targets, dtype=int).clip(min=0))
                 for crop, label in zip(crops, list(text_targets)):
-                    self.data.append((crop, dict(labels=[label])))
+                    # filter labels with whitespaces
+                    if ' ' not in label:
```

Comment:
just to make sure: what is the expected behaviour here?

Because I get the feeling that:
- it was made to delete labels that are white spaces
- but here it will delete label that have whitespaces (e.g. "Hello world")

I'm not saying it's something we can easily process but it's good to be transparent on that part

Reply:
Ah damn, i was a bit to much focused on the behavior we need inside docTR but yes i agree we cannot do this will be changed !

And no it was exactly made to avoid every label like this: "Hello World" the dataset does not contains empty labels